Updated link to new docs site

JSchripsema-IDM opened the pull request.

Commit:
Updated link to new docs site

clorton commented on README.md:
```diff
@@ -10,4 +10,4 @@ This repository uses [LFS](https://git-lfs.github.com/) (large file storage) to
 *Additional Note:* the GitHub "Download .ZIP" button will not package and deliver the actual binary data of the LFS managed files. You will need to use Git and follow the steps listed above in order to download the input data files.
 
-<a href="https://zenhub.com"><img src="https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.png"></a>
+For more information on downloading and using these files, see the EMOD software documentation at https://institutefordiseasemodeling.github.io/EMOD/.
```
I think we agreed to have a ZenHub badge on each of our public projects using ZenHub. Please add the badge link back in (in addition to the new documentation link).

Commit:
Update README.md

Ah! Apologies. I only saw it on repos that had pretty minimal READMEs so I assumed it was automatically added by Github and not removing it was an oversight.

clorton commented on README.md:
```diff
@@ -10,4 +10,6 @@ This repository uses [LFS](https://git-lfs.github.com/) (large file storage) to
 
 *Additional Note:* the GitHub "Download .ZIP" button will not package and deliver the actual binary data of the LFS managed files. You will need to use Git and follow the steps listed above in order to download the input data files.
 
+For more information on downloading and using these files, see the EMOD software documentation at https://institutefordiseasemodeling.github.io/EMOD/.
```
This information should be a live link:

```[https://institutefordiseasemodeling.github.io/EMOD/](https://institutefordiseasemodeling.github.io/EMOD/)```

[https://institutefordiseasemodeling.github.io/EMOD/](https://institutefordiseasemodeling.github.io/EMOD/)

Commit:
Fixed link in README.
